frontend/app/professor/simulation-builder/page.tsx

```
@@ -1981,258 +2234,273 @@ return (
+                     : 'border-gray-300 bg-card hover:border-gray-400'
+                 }`}
+                 onDragOver={handleDragOver}
+                 onDragLeave={handleDragLeave}
+                 onDrop={handleDrop}
+                 onClick={() => fileInputRef.current?.click()}
+               >
+                 {uploadedFile ? (
+                   <span className="flex flex-col items-center">
+                     {/* Red file icon */}
+                     <svg className="h-10 w-10 mx-auto mb-2 text-red-500" fill="none" stroke="currentColor" strokeWidth="1.5" viewBox="0 0 24 24">
+                       <path d="M14 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V8z" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)